pkg/pods/manifest.go

```diff
@@ -41,6 +41,14 @@ type Manifest struct {
 	signature []byte
 }
 
+type JSONManifest struct {
```

Comment:
Hmm. I was about to ask "Can we embed JSONManifest into Manifest so we don't forget and accidentally get them out of sync with each other?" but then I saw the json annotations and realized it may not be possible. Unless it's possible to have both a json annotation and a yaml annotation on a struct member...? I'd at least investigate the possibility. If not, at least to comment on both structs saying they need to be kept in sync.